### ISSUE TITLE
Additional tiling of the plane

### ISSUE THREAD

**Issue by yashton** [yashton/terpstrakeyboard#22]
Hexagons aren't the only tilings available: squares, rectangles, parallelograms and rhombuses in the quadrilaterals Hexagons can also be stretched. This would maintain the isomorphism with maybe providing interesting options or better ergonomics.

**Comment by pierstitus** [comment]
I totally agree on this! Apparently there's already so much to discover that people forget to look beyond  hexagonal or square layouts.

It would be very nice to be able to be able to play the layout I developed on this web app! See www.striso.org.

I also made a web app for this layout, but didn't yet make it playable with mouse or touchscreen, see www.toverlamp.org/static/wickisynth/wickisynth.html
